README.rst

```diff
@@ -156,7 +156,7 @@ A one bin example
    import pyhf
```

Comment:
should the images in the README also be updated to the new ones (w/ legends)

Reply:
yup! They are.

```console
$ git diff origin/master --name-only | grep static
docs/_static/img/README_1bin_example.png
docs/_static/img/README_2bin_example.png
```

c.f. the "rich diff" view on the images in the "Files changed" tab to see the difference rendered out.

Reply:
yeah I see thoses, but looking at

https://github.com/scikit-hep/pyhf/blob/7ec11ae45ca1053b27ca23ac2069ec36a65d3ba2/README.rst

I don't see it

Reply:
Oh yeah, that's because we have it pinned to the `master` branch files in the README so that it shows up properly other places on the web (I think that's the reason at least).

```console
$ grep README_1bin_example README.rst 
.. image:: https://raw.githubusercontent.com/scikit-hep/pyhf/master/docs/_static/img/README_1bin_example.png
```